core/foundation/src/TError.cxx

```diff
@@ -66,6 +66,7 @@ ErrorSystemMsgHandlerFunc_t SetErrorSystemMsgHandler(ErrorSystemMsgHandlerFunc_t
 /// A very simple error handler that is usually replaced by the TROOT default error handler.
 /// The minimal error handler is not serialized across threads, so that output of multi-threaded programs
 /// can get scrambled
+/// @note `abort()` is only called if `abort_bool` is `true` and `level < gErrorIgnoreLevel`
```

Comment:
Not true :slightly_smiling_face:; if `level < gErrorIgnoreLevel`, the function exits immediately (see line 73).

Reply:
Fixed :) thks